### PR TITLE
[CodeMirror] Add commands actions

### PR DESCRIPTION
**Pull request by ficristo**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://codemirror.net/doc/manual.html
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/codemirror/index.d.ts [hunk 1]
```diff
@@ -1222,6 +1222,153 @@ declare namespace CodeMirror {
      */
     function overlayMode<T, S>(base: Mode<T>, overlay: Mode<S>, combine?: boolean): Mode<any>;
 
+    interface CommandActions {
+        /** Select the whole content of the editor. */
+        selectAll(): void;
```
Comment (mihailik) [inline types/codemirror/index.d.ts]: Shouldn't this have `cm: CodeMirror.Editor` argument too?

**Review by mihailik — CHANGES_REQUESTED** [review]

File: types/codemirror/index.d.ts (hunk 1 above)
Comment (ficristo) [reply types/codemirror/index.d.ts]: My bad... Fixed.

**Comment by ficristo** [comment]
@mihailik could you take another look?

**Review by mihailik — APPROVED** [review]
Good stuff, sorry for being a bit of a slow poke on this one.

**Comment by ficristo** [comment]
@mihailik no problem and thank you.

**Pull request merged**